manifests/install.pp

```diff
@@ -60,7 +60,7 @@
     file { 'kibana-init-script':
       path    => '/etc/init.d/kibana',
       ensure  => 'file',
-      content => template('kibana/kibana.legacy.service.erb'),
+      content => template("kibana/$::kibana::params::init_script_template"),
```

Comment:
Variables interpolated in strings needs to be written as "blah blah ${var name} blah blah"  (need the curly braces around the variable) to keep puppet happy.